go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	github.com/cosmos/cosmos-sdk v0.46.11
 	github.com/cosmos/gogoproto v1.4.2
 	github.com/cosmos/ibc-go/v6 v6.1.0
-	github.com/rs/zerolog v1.27.0
+	github.com/rs/zerolog v1.29.1
```

Comment:
v1.27.0 is what is used by cosmos-sdk v0.46.11 so we can close this PR

Ref: https://github.com/cosmos/cosmos-sdk/blob/5023392bd9718679668dd8e88a792a6defa63c00/go.mod#L44